lib/crawly/worker.ex

```diff
@@ -76,16 +76,26 @@ defmodule Crawly.Worker do
              spider_name: atom(),
              response: HTTPoison.Response.t(),
              result: {:ok, {response, spider_name}} | {:error, term()}
-  defp get_response({request, spider_name}) do
+  def get_response({request, spider_name}) do
     # check if spider-level fetcher is set. Overrides the globally configured fetcher.
     # if not set, log warning for explicit config preferred,
     # get the globally-configured fetcher. Defaults to HTTPoisonFetcher
+
     {fetcher, options} =
-      Crawly.Utils.get_settings(
-        :fetcher,
-        spider_name,
-        {Crawly.Fetchers.HTTPoisonFetcher, []}
-      )
+      case Crawly.Utils.get_settings(
+             :fetcher,
+             spider_name,
+             {Crawly.Fetchers.HTTPoisonFetcher, []}
+           ) do
+        {module, args} ->
+          {module, args}
+
+        {module} ->
```

Comment:
this format is not supported, should not handle it. should throw an error if user attempts to use this format.

Reply:
please remove this case.

Reply:
Done